srcpkgs/flatbuffers/template

```diff
@@ -19,7 +19,7 @@ case "$XBPS_TARGET_MACHINE" in
 	*-musl) configure_args+=" -DFLATBUFFERS_LOCALE_INDEPENDENT=0";;
 esac
 
-CXXFLAGS="-Wno-class-memaccess -Wno-stringop-overflow"
+CXXFLAGS="-O"
```

Comment:
?

Reply:
Fortify source level 2 that is enabled by default by void build scripts requires that code is compiled with optimization enabled.
But maybe there is something that I have misunderstood?

Reply:
Can you share those errors?

In any case, `-O` is just plain wrong...